BaseProject/WebContent/index.htm

```diff
@@ -27,8 +27,10 @@
                   <li class="dropdown">
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Employee <span class="caret"></span></a>
                      <ul class="dropdown-menu">
+
                         <li><a href="./ShiftServlet">Search Shifts</a></li>
-			<li><a href="./insertProduct">Insert new product</a></li>
+			                  <li><a href="./insertProductServlet">Insert new product</a></li>
```

Comment:
For me this doesn't work. The servlet annotation is `InsertProductServlet` but it's `insertProductServlet` here. I think it should be capital.